### PR TITLE
Removing the Daml Ledgers table

### PR DESCRIPTION
**Pull request by carrielaben-da**
Removes the existing Deploying to Daml Ledgers table and moves Deploying To A Generic Daml Ledger up in the TOC

[CHANGELOG_BEGIN]

[CHANGELOG_END]

### CODE REVIEW

**Review by garyverhaegen-da — APPROVED** [review]
Thanks!

File: docs/source/index.rst [hunk 1]
```diff
@@ -47,7 +47,7 @@ Daml Documentation
    :hidden:
    :caption: Deploying to Daml ledgers
 
-   deploying
+   deploy/generic_ledger
```
Comment (garyverhaegen-da) [inline docs/source/index.rst:50]: You probably need to reflect that change in the PDF one too.

Comment (carrielaben-da) [reply docs/source/index.rst:50]: Thanks, added that.

**Comment by garyverhaegen-da** [comment]
There seems to be a link to `/deploying` at the top of the `docs/source/daml/reference/packages.rst` page, that needs to be removed (probably with a little bit of rewriting; I haven't looked at the context very closely).

**Pull request merged**